src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,28 @@
 import React from 'react';
+// import commentsFromServer from '../../api/comments';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  name: string,
+  email: string,
+  body: string;
```

Comment:
Consider replacing the semicolon with a comma in the type definition for better consistency 🔄